src/constants/index.ts

```diff
@@ -94,7 +94,7 @@ export const CLOUD_8 = "./assets/images/cloud_03.png";
 
 export const PLAY_BTN_IMG = "./assets/images/Play_button.webp";
 export const PAUSE_BTN_IMG = "./assets/images/pause_v01.webp";
-export const CLOSE_BTN_IMG = "./assets/images/map_btn.webp";
+export const CLOSE_BTN_IMG = "./assets/images/close_btn.webp";
```

Comment:
keep the file name same as before just not to misunderstand

Reply:
fixed the issue on 
![image](https://github.com/user-attachments/assets/946a20b0-49c4-45af-b8d6-9e2385a6ae47)